docs/guide/images.md

```diff
@@ -225,3 +225,112 @@ The following filters exist in the Imagine library but are not used in eZ Platfo
 ### Custom filters
 
 Please refer to [LiipImagineBundle documentation on custom filters](http://symfony.com/doc/master/bundles/LiipImagineBundle/filters.html#custom-filters). [Imagine library documentation](http://imagine.readthedocs.org/en/latest/) may also be useful.
+
+## Setting placeholder generator
+
+Placeholder generator enables you to download or use predefined image placeholder for any missing image.
+
+`PlaceholderAliasGenerator::getVariation` method generates placeholder (by delegating it to the implementation of `PlaceholderProvider` interface) if original image cannot be resolved and saves it under the original path.
+
+!!! note 
+
+    Width and height of an original image are required to generate proper placeholder. `PlaceholderAliasGenerator` decorates `AliasGenerator` as `AliasGenerator` is the only place where you can find them.  
+
+This chapter includes two implementations of `PlaceholderProvider` interface:
```

Comment:
Not sure about "chapter" (I think we'd better avoid such 'meta' descriptions of the text). Maybe something like this instead: `Two implementations of PlaceholderProvider are used to...`